govcd/system.go

```diff
@@ -337,6 +365,37 @@ func getOrgHREF(vcdClient *VCDClient, orgName string) (string, error) {
 	return "", fmt.Errorf("couldn't find org with name: %s. Please check Org name as it is case sensitive", orgName)
 }
 
+// Returns the HREF of the org from the org ID
+func getOrgHREFById(vcdClient *VCDClient, orgId string) (string, error) {
+	orgListHREF := vcdClient.Client.VCDHREF
```

Comment:
Maybe it's possible to Query using href/id in filter?

Reply:
Query struct doesn't have Organization info.

Reply:
Yes, It would be need to add.